v-next/hardhat/src/internal/builtin-plugins/network-manager/edr/stack-traces/stack-trace-solidity-errors.ts

```diff
@@ -0,0 +1,428 @@
+import type {
+  SolidityStackTrace,
+  SolidityStackTraceEntry,
+  SourceReference,
+} from "./solidity-stack-trace.js";
+
+import { bytesToHexString as bufferToHex } from "@ignored/hardhat-vnext-utils/bytes";
+import { ReturnData } from "@nomicfoundation/edr";
+
+import { panicErrorCodeToMessage } from "./panic-errors.js";
+import {
+  CONSTRUCTOR_FUNCTION_NAME,
+  PRECOMPILE_FUNCTION_NAME,
+  StackTraceEntryType,
+  UNKNOWN_FUNCTION_NAME,
+  UNRECOGNIZED_CONTRACT_NAME,
+  UNRECOGNIZED_FUNCTION_NAME,
+} from "./solidity-stack-trace.js";
+
+export function encodeSolidityStackTrace(
+  fallbackMessage: string,
+  stackTrace: SolidityStackTrace,
+  previousStack?: NodeJS.CallSite[],
+): SolidityError {
+  if (Error.prepareStackTrace === undefined) {
```

Comment:
I suspect this should be removed.

Reply:
Yes, this can be removed. 22.0.0 already includes it.